structuredheadings/styles/numbering.css

```diff
@@ -26,6 +26,10 @@ h6.autonumber {
 	counter-increment: h6;
 }
 
+h1.autonumber-restart {
+	counter-reset: h1;
+}
```

Comment:
I also wonder a bit about the virtues of this over having some separate element that restarts things (like a section break.)  I haven't given it much thought though.